static/app/views/settings/account/apiNewToken.tsx

```diff
@@ -46,31 +61,30 @@ export default class ApiNewToken extends Component {
             )}
           </TextBlock>
           <Panel>
-            <PanelHeader>{t('Create New User Auth Token')}</PanelHeader>
+            <PanelHeader>{t('Permissions')}</PanelHeader>
             <ApiForm
               apiMethod="POST"
               apiEndpoint="/api-tokens/"
-              initialData={{scopes: SORTED_DEFAULT_API_ACCESS_SCOPES}}
+              initialData={{scopes: []}}
               onSubmitSuccess={this.onSubmitSuccess}
               onCancel={this.onCancel}
               footerStyle={{
                 marginTop: 0,
                 paddingRight: 20,
               }}
+              submitDisabled={Object.values(this.state.permissions).every(
```

Comment:
nit: pull this out into its own const like

```
const {permissions} = this.state;
...
{Object.values(permissions).every(...
```

and then below when you pass `this.state.permissions` again you can just use `permissions`